### PR TITLE
fix: failed upload cli

### PR DESCRIPTION
**Pull request by thxCode**
## problem

for now, the package layout looks as shown, which causes two problems: 1. UI cannot download CLI as not found; 2. failed to upload CLI to Tencent COS.

image

## solution

make sure the package layout is as below:
image

- upload assets to COS  from `walrus/server/cos` dir.
- release assets to Github from `walrus/server/release` dir.

## related
#2080

### CODE REVIEW

File: hack/package.sh [hunk 1]
```diff
@@ -96,6 +139,9 @@ function package() {
   # shellcheck disable=SC2155
   local context="$(setup_image_package_context "${target}" "${task}" "${path}")"
 
+  setup_release "${context}"
+  setup_tencent_cos "${context}"
```
Comment (shanewxy) [inline hack/package.sh:143]: Do we need to check `PACKAGE_BUILD=false` before setup release?

Comment (thxCode) [reply hack/package.sh:143]: that's ok, we just do these steps before PACKAGE_BUILD.

**Review by shanewxy — APPROVED** [review]
LGTM

**Review by aiwantaozi — APPROVED** [review]
LGTM

**Pull request merged**